structuredheadings/plugin.js

```diff
@@ -1,10 +1,11 @@
-/*eslint max-statements: [2, 25]*/ //I'll need to refactor somehow to meet this rule
+/*eslint max-statements: [2, 50]*/ //I'll need to refactor somehow to meet this rule
```

Comment:
Yep, tired of it :)
BUT... I have spent some time looking around at the way some other plugins are structured (MOST seem to throw everything under init just like I am) and I think I maybe found another way of doing things, I'm going to try it.